spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,47 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
+<<<<<<< HEAD
+=======
```

Comment:
Bad line breaking before '==='.<br>Expected an identifier and instead saw '==='.<br>Unrecoverable syntax error. (79% scanned).